lightning/src/ln/functional_tests.rs

```diff
@@ -7298,3 +7298,22 @@ fn test_bump_txn_sanitize_tracking_maps() {
 		}
 	}
 }
+
+#[test]
+fn test_override_channel_config() {
+    let chanmon_cfgs = create_chanmon_cfgs(2);
```

Comment:
nit: Same here, please use indentation characters instead of spaces.

Reply:
fixed both, perhaps better to work on the requested updates to the tests before squashing? - might be easier for everyone to match the review requested per commit